synapse/tests/test_lib_modelrev.py

```diff
@@ -473,3 +473,26 @@ async def test_modelrev_0_2_24(self):
             nodes = await core.nodes('it:mitre:attack:technique=T0100')
             self.len(1, nodes)
             self.eq('lockpicking', nodes[0].get('name'))
+
+    async def test_modelrev_0_2_25(self):
+        async with self.getRegrCore('model-0.2.25') as core:
+
+            nodes = await core.nodes('ou:conference')
+            self.len(5, nodes)
+            names = [n.get('name') for n in nodes]
+            self.sorteq(names, (
+                'sleuthcon',
+                'defcon',
+                'recon 2024 conference',
+                'blackhat',
+                'summercon',
+            ))
+
+            nodes = await core.nodes('ou:position')
+            self.len(3, nodes)
+            titles = [n.get('title') for n in nodes]
+            self.sorteq(titles, (
+                'president of the united states',
+                'vice president of the united states',
+                'speaker of the house, united states congress',
+            ))
```

Comment:
Need tests that confirm the `ou:jobtitle` and `entity:name` forms have also been created and are pivotable :+1: